src/app/profile/overview/work-items/work-items.component.ts

```diff
@@ -73,122 +67,63 @@ export class WorkItemsComponent implements OnDestroy, OnInit  {
     });
   }
 
-  /**
-   * Fetch work items
-   */
-  fetchWorkItems(): void {
-    this.fetchWorkItemsBySpace(this.getSpaceById(this.currentSpaceId));
-  }
-
-  /**
-   * Fetch space for current space ID
-   *
-   * @returns {Observable<Space>}
-   */
-  get space(): Observable<Space> {
-    return this.userService.loggedInUser
-      .map(user => this.spaceService.getSpacesByUser(user.attributes.username, 10))
-      .switchMap(spaces => spaces)
-      .map(spaces => {
-        for (let i = 0; i < spaces.length; i++) {
-          if (this.currentSpaceId === spaces[i].id) {
-            return spaces[i];
-          }
+  private mergeSpaces(spaces: Space[], recentSpaces: Space[]): void {
+    let found: boolean;
+    for (let i: number = 0; i < recentSpaces.length; i++) {
+      found = false;
+      for (let j: number = 0; j < spaces.length; j++) {
+        if (recentSpaces[i].id === spaces[j].id) {
+          found = true;
+          break;
```

Comment:
Bad rebase? `mergeSpaces` has come back again

Reply:
Dang. I wonder what I did wrong. Anyways I reworked the patch series again. It should be okay now. Sorry about that.